apps/frontend/src/components/Surface.tsx

```diff
@@ -13,8 +13,8 @@ export const Surface = forwardRef<HTMLDivElement, SurfaceProps>(
     ref
   ) => {
     const surfaceClass = cn(
-      className,
       "bg-white rounded-xl dark:bg-neutral-900",
+      className,
       withShadow ? "shadow-sm" : "",
       withBorder ? "border border-neutral-200 dark:border-neutral-800" : ""
```

Comment:
```suggestion
      withBorder ? "border border-border" : ""
```

also, we have a color variable called "border" so we can use border-border and it will be applied for both light and dark mode.